From: Joe Nahmias <jello@debian.org>
Date: Fri, 28 Jun 2013 00:00:00 +0000
Subject: fix manpage to use minus sign instead of hyphens

Description: fix manpage to use minus sign instead of hyphens
Author: Joe Nahmias <jello@debian.org>
Last-Update: 2013-06-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 xsel.1x | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/xsel.1x b/xsel.1x
index 3f16f5b..fe64460 100644
--- a/xsel.1x
+++ b/xsel.1x
@@ -105,7 +105,7 @@ fork to become a background process in \fIinput\fR, \fIexchange\fR and
 display usage information and exit
 .TP
 \fB\-v\fR, \fB\-\-verbose\fR
-Print informative messages. Additional instances of \fI-v\fR raise the
+Print informative messages. Additional instances of \fI\-v\fR raise the
 debugging level, ie. print more information.
 .TP
 \fB\-\-version\fR
@@ -121,10 +121,10 @@ mouse button paste. In order to implement modification of the selection(s)
 from the terminal, spawning a child process to supply the new selection(s)
 on demand. This child exits immediately when any other program takes over
 the selection(s), eg. when the user next selects some text in a terminal
-window or by running \fBxsel -c\fR.
+window or by running \fBxsel \-c\fR.
 .PP
 .SH STANDARDS
-xsel conforms to the Inter-Client Communication Conventions Manual
+xsel conforms to the Inter\-Client Communication Conventions Manual
 Version 2.0 (ICCCM2).
 .PP
 .SH "SEE ALSO"
